projects/laji/src/app/shared-modules/spreadsheet/importer/cell-value-mapping/cell-value-select/cell-value-select.component.ts

```diff
@@ -16,7 +17,8 @@ export class CellValueSelectComponent {
   _field: IFormField;
   labels: string[] = [];
   ignore = VALUE_IGNORE;
-  booleanValues = [VALUE_IGNORE, 'true', 'false'];
+  booleanValueMap: {[key: string]: typeof VALUE_IGNORE|boolean} = {};
+  booleanValues: string[] = [];
```

Comment:
`booleanValues` is unused

Reply:
The template is using it

Reply:
Oh sorry, I read this as an added property but it's a modification.